src/NATS.Client.Core/Internal/netstandard.cs

```diff
@@ -10,9 +10,13 @@
 #pragma warning disable SA1204
 #pragma warning disable SA1405
 
-#if NETSTANDARD
-
 // Enable init only setters
+#if NET5_0_OR_GREATER
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.IsExternalInit))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.CallerArgumentExpressionAttribute))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Runtime.CompilerServices.SkipLocalsInitAttribute))]
```

Comment:
Looks like `CallerArgumentExpression` and `SkipLocalsInit` are only required by the compiler and don't need to be forwarded. I can reproduce the issue when not having `IsExternalInit` but other two doesn't make any difference. Also redis library only has`IsExternalInit` forwarded. Do you think we need the other two?

Reply:
Redis only has an implementation for IsExternalInit, which is why it only has a forwarder for that type.

Here we have implementations for:
`IsExternalInit`
`CallerArgumentExpressionAttribute`
`SkipLocalsInitAttribute`

I'm unsure where those are used in the codebase, but if used in any codepath similar to the issue that I noticed with `IsExternalInit`, (if I understand the root cause of this correctly...) they won't exist.

Happy to remove those if preferred though since I don't know a way to break things with those forwarders missing

Reply:
New commit incoming